src/components/Cards/index.tsx

```diff
@@ -1,9 +1,10 @@
-import { Grid, Stack, Typography } from '@mui/material';
+import { Button, Grid, Stack, Typography } from '@mui/material';
 import { type Generic, type CommonProps } from '../../types/components';
 import EContainer from '../EContainer';
 import Item, { type IItem } from './item';
 import { isJSX } from '../../utils';
 import { type ReactNode } from 'react';
+import { type CtaButton } from 'components/Ctas';
```

Comment:
Are you sure about this import? Is the proper syntax?

Reply:
for this import, I used the classic one but eslint told me that was an issue and let me change to this other version

Reply:
ok, but I'm not sure about the `components` part.

I think `import { type CtaButton } from '../Ctas';` works as well and is coherent with other imports